src/entries/popup/pages/home/NFTs/SortDropdown.tsx

```diff
@@ -26,36 +28,57 @@ export default function SortdDropdown() {
     [setNftSort],
   );
   const { currentTheme } = useCurrentThemeStore();
+  const [open, setIsOpen] = useState(false);
+
+  useKeyboardShortcut({
+    condition: () => open,
+    handler: (e) => {
+      e.stopImmediatePropagation();
+      if (e.key === shortcuts.home.NFT_SORT_RECENT.key) {
+        onValueChange('recent');
+        setIsOpen(false);
+      } else if (e.key === shortcuts.home.NFT_SORT_ABC.key) {
+        onValueChange('alphabetical');
```

Comment:
Should we move all these strings as const to prevent typos?

Reply:
The values are typed, so we get autocomplete and if there's a typo TS will complain.